include/arduino-mock/serial.h

```diff
@@ -91,4 +100,26 @@ class Serial_ {
 SerialMock* serialMockInstance();
 void releaseSerialMock();
 
+
+/*
+ * Thsi is a helper class which can be Invoked on EXPECT_CALL to gather data
+ * from repeated calls to Serial functions. For example usage, see the unit
+ * tests in test/serial_unittest.cc
+ *
+ */
+class stringCapture {
```

Comment:
Hi Ikeasu, I wrote a longer response below. The stringCapture stuff is supposed to be used by external testers. The naming is a bit bad, any thoughts on a new name?